[2024-07-22T16:43:11.966+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-22T16:43:12.004+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_complex.delete_entry manual__2024-07-22T16:41:58.966161+00:00 [queued]>
[2024-07-22T16:43:12.020+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_complex.delete_entry manual__2024-07-22T16:41:58.966161+00:00 [queued]>
[2024-07-22T16:43:12.021+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-22T16:43:12.039+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): delete_entry> on 2024-07-22 16:41:58.966161+00:00
[2024-07-22T16:43:12.049+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=245) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-22T16:43:12.051+0000] {standard_task_runner.py:63} INFO - Started process 247 to run task
[2024-07-22T16:43:12.052+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_complex', 'delete_entry', 'manual__2024-07-22T16:41:58.966161+00:00', '--job-id', '39', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_complex.py', '--cfg-path', '/tmp/tmpbfvcbhb4']
[2024-07-22T16:43:12.055+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask delete_entry
[2024-07-22T16:43:12.121+0000] {task_command.py:426} INFO - Running <TaskInstance: example_complex.delete_entry manual__2024-07-22T16:41:58.966161+00:00 [running]> on host 1398923a395d
[2024-07-22T16:43:12.229+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_complex' AIRFLOW_CTX_TASK_ID='delete_entry' AIRFLOW_CTX_EXECUTION_DATE='2024-07-22T16:41:58.966161+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-22T16:41:58.966161+00:00'
[2024-07-22T16:43:12.232+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-22T16:43:12.233+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-22T16:43:12.234+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo delete_entry']
[2024-07-22T16:43:12.251+0000] {subprocess.py:86} INFO - Output:
[2024-07-22T16:43:12.254+0000] {subprocess.py:93} INFO - delete_entry
[2024-07-22T16:43:12.254+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-22T16:43:12.255+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-22T16:43:12.288+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=delete_entry, run_id=manual__2024-07-22T16:41:58.966161+00:00, execution_date=20240722T164158, start_date=20240722T164312, end_date=20240722T164312
[2024-07-22T16:43:12.346+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-22T16:43:12.371+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-22T16:43:12.373+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
